src/components/SearchVaccines.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import styled from "styled-components";
+
+const TextField = styled.input`
+  display: flex;
+  flex-direction: column;
+  width: 330px;
+  height: 45px;
+  border-radius: 20px;
+  background-color: white;
+  border: #707070 solid 1px;
+  opacity: 0.5;
+  margin-top: 10px;
+  text-align: center;
+
+  &::placeholder {
+    color: grey;
+  }
+`;
+
+export default function SearchVaccines({ onSearch }) {
```

Comment:
Sounds like a function but not like a component to me.

Reply:
SearchBar sounds like a component for example.

Reply:
done :)